src/main/steps/applicant1/equality/createToken.ts

```diff
@@ -0,0 +1,32 @@
+import crypto from 'crypto';
+
+import { Logger } from '@hmcts/nodejs-logging';
+import config from 'config';
+
+const logger = Logger.getLogger('createToken');
+
+const algorithm = 'aes-256-gcm';
+const bufferSize = 16;
+const iv = Buffer.alloc(bufferSize, 0); // Initialization vector.
+const keyLen = 32;
+
+export const createToken = (params: Record<string, unknown>): string => {
```

Comment:
Can we do better than `Record<string, unknown>` here? I'm not really sure what params is supposed to be

Reply:
Yeah can create a PCQ params type or something like that